concert/tests/util/focus.py

```diff
@@ -11,7 +14,7 @@ class BlurringCamera(DummyCameraBase):
 
     async def __ainit__(self, motor):
         await super(BlurringCamera, self).__ainit__()
-        self._original = scipy.misc.ascent()
```

Comment:
At least put there try-except and do it the old way in case scipy is old, please.